cmd/subctl/verify.go

```diff
@@ -62,18 +62,21 @@ var (
 
 var verifyRestConfigProducer = restconfig.NewProducer().
 	WithPrefixedContext("to").
+	WithPrefixedContext("extra").
 	WithDefaultNamespace(constants.OperatorNamespace)
 
 var verifyCmd = &cobra.Command{
-	Use:   "verify --context <kubeContext1> --tocontext <kubeContext2>",
+	Use:   "verify --context <kubeContext1> --tocontext <kubeContext2> [--extracontext <kubeContext3>]",
 	Short: "Run verifications between two clusters",
-	Long: `This command performs various tests to verify that a Submariner deployment between two clusters
-is functioning properly. The verifications performed are controlled by the --only and --enable-disruptive
-flags. All verifications listed in --only are performed with special handling for those deemed as disruptive.
-A disruptive verification is one that changes the state of the clusters as a side effect. If running the
-command interactively, you will be prompted for confirmation to perform disruptive verifications unless
-the --enable-disruptive flag is also specified. If running non-interactively (that is with no stdin),
---enable-disruptive must be specified otherwise disruptive verifications are skipped.
+	Long: `This command performs various tests to verify that a Submariner deployment between two clusters,
+specified via the --context and --tocontext args, is functioning properly. Some tests require a third cluster,
```

Comment:
How about specifying "Some servicediscovery tests require a third cluster" to make it clear that if someone is not using servicediscovery verify they don't need to use this argument.

Or, if service discovery is not enabled for verification give a warn or info message to user that `--extracontext` will be ignored.

Reply:
I was thinking not to make it specific to service discovery in case we ever add other test(s) that need a third cluster.

Reply:
Making it genric may be a good idea as there could be use cases later for newer test in connectivity requiring a third cluster.

Reply:
> I was thinking not to make it specific to service discovery in case we ever add other test(s) that need a third cluster.

We can change the info message when we need it for other suites. As of now it is only service discovery, will help user know that this will apply only to service discovery and other tests don't need this option.

Reply:
yeah I agree it would be useful. We can change it later if needed although that kind of thing tends to slip through the cracks.

Reply:
Done